.github/workflows/ci.yml

```diff
@@ -20,39 +20,59 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
         include:
           - framework: tensorflow
```

Comment:
It may make sense to have a separate attribute here called `name` where we can give a short name to each test variant, so it makes sense when debugging. Ideally we wouldn't have two tests that report with the same name. To that end we could also probably drop the `tf_version` attribute

Reply:
I think that makes sense, I'll add it in new commits.